components/sections/Footer/index.tsx

```diff
@@ -0,0 +1,58 @@
+import classNames from 'classnames';
+import Image from 'next/image';
+import type { FC } from 'react';
+import { FormattedMessage } from 'react-intl';
+
+import NavItem from '@/components/sections/NavItem';
+import { useSiteConfig } from '@/hooks/useSiteConfig';
+
+import styles from './index.module.css';
+
+const Footer: FC = () => {
+  const { footerLinks, socialLinks } = useSiteConfig();
+
+  const openJSlink = footerLinks.at(-1)!;
```

Comment:
Not sure if this is the best approach here, but we can change in the future if needed.